### ISSUE TITLE
ESRI expects WMS mimetype text/xml, not happy with application/vnd.ogc.wms_xml

### ISSUE THREAD

**Issue by nigels-com** [mapproxy/mapproxy#362]
We're having an issue that appears to be along the lines [discussed here](https://community.esri.com/thread/85095).  Mapproxy is returning **application/vnd.ogc.wms_xml** as the mimetype for `service?SERVICE=WMS&REQUEST=GetCapabilities` but ESRI applications are expecting a mimetype of **text/xml**.

According to the [OpenGIS® Web Map Server Implementation Specification 1.3.0](http://portal.opengeospatial.org/files/?artifact_id=14416) section 7.2.3.1 "Every server shall support the default text/xml format..." and " If the request specifies a format not supported by the server, the server shall respond with the default text/xml format. ".

## Steps to Reproduce

For our mapproxy deployment:

~~~~
services:
  kml:
    use_grid_names: true
  wmts:
  wms:
    srs: ['EPSG:3857', 'EPSG:4326', 'CRS:84', 'EPSG:32755', 'EPSG:32756', 'EPSG:28355', 'EPSG:28356']
    md:
      title: Imagery
      abstract: Imagery
~~~~

~~~~
wget 'https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities'
--2018-04-13 17:47:03--  https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities
Resolving ...
Connecting to ... connected.
HTTP request sent, awaiting response... 200 OK
Length: 14188 (14K) [application/vnd.ogc.wms_xml]
~~~~

Same result for FORMAT=text/xml:

~~~~
wget 'https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities&FORMAT=text/xml'
~~~~

For VERSION=1.3.0 it is according to spec:

~~~~
wget 'https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.3.0'
...
HTTP request sent, awaiting response... 200 OK
Length: 22804 (22K) [text/xml]
~~~~

And... VERSION=2.0.0 is `text/xml`

~~~~
wget 'https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=2.0.0'
...
HTTP request sent, awaiting response... 200 OK
Length: 22804 (22K) [text/xml]
~~~~

And it gets interesting with `FORMAT=foo`

~~~~
wget 'https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.2.0&FORMAT=foo'
...
HTTP request sent, awaiting response... 200 OK
Length: 14188 (14K) [application/vnd.ogc.wms_xml]
~~~~

~~~~
wget 'https://wms.foo.com/service?SERVICE=WMS&REQUEST=GetCapabilities&VERSION=1.3.0&FORMAT=foo'
...
HTTP request sent, awaiting response... 200 OK
Length: 22804 (22K) [text/xml]
~~~~

Which does appear to be according to spec for 1.3.0.

## Discussion

I appreciate here that there are likely to be downstream consequences to a switch-over to `text/xml` but in our case we'd at least like to configure Mapproxy to use the ESRI-friendly (and standards based?) mimetype.

If you can provide some guidance I'd be happy to put together a pull request to our mutual satisfaction.

For our purposes it would probably suffice to default to `VERSION=1.3.0` for the purpose of `service?SERVICE=WMS&REQUEST=GetCapabilities`

Alternatively we can configure Mapproxy to only support `VERSION=1.3.0`:

~~~~
  wms:
    versions: [ '1.3.0' ]
~~~~

**Comment by nigels-com** [comment]
I noticed today that there was a fix for this issue committed since the last release:
https://github.com/mapproxy/mapproxy/commit/c84322bc7812b435a9bb71e306da4f35a2af875d

This appears to be a good solution to this problem, defaulting to 1.3.0 and `text/xml` mime type.

**Comment by olt** [comment]
This is likely a bug in your ESRI software. `application/vnd.ogc.wms_xml` is the right content-type according to the WMS 1.1.1 spec. 

Anyway, this is "solved" with c84322b.
